source/php/Util/PostQuery.php

```diff
@@ -24,7 +24,7 @@ private static function userSelectQuery() {
     $postName = self::$postTbName;
     if (Settings::getUseAlternateUserField()) {
       global $wpdb;
-      return "(SELECT meta_value FROM $wpdb->postmeta WHERE post_id = $postName.ID AND meta_key LIKE 'page_meta_maineditor' AND post.post_type = 'page') = user.ID OR (post.post_author = user.ID AND post.post_type != 'page')";
+      return "(SELECT meta_value FROM $wpdb->postmeta WHERE post_id = $postName.ID AND meta_key LIKE 'page_meta_maineditor') = user.ID OR (post.post_author = user.ID)";
```

Comment:
Noticed that if the author is A and editor is set to B on a page or project, that page or project will appear in the list for both A and B. With the suggested change we will only fall back on author if there is no editor set on the post.
```suggestion
      return "(SELECT meta_value FROM $wpdb->postmeta WHERE post_id = $postName.ID AND meta_key LIKE 'page_meta_maineditor') = user.ID OR ((SELECT count(1) FROM $wpdb->postmeta WHERE post_id = $postName.ID AND meta_key LIKE 'page_meta_maineditor') = 0 AND post.post_author = user.ID)";
```